pkg/cmd/repo/create/create.go

```diff
@@ -126,67 +180,185 @@ func createRun(opts *CreateOptions) error {
+				if err != nil {
+					return err
+				}
+			}
+			if doSetup {
+				path := repo.Name
+
+				gitInit := git.GitCommand("init", path)
+				gitInit.Stdout = stdout
+				gitInit.Stderr = stderr
+				err = run.PrepareCmd(gitInit).Run()
+				if err != nil {
+					return err
+				}
+				gitRemoteAdd := git.GitCommand("-C", path, "remote", "add", "origin", remoteURL)
+				gitRemoteAdd.Stdout = stdout
+				gitRemoteAdd.Stderr = stderr
+				err = run.PrepareCmd(gitRemoteAdd).Run()
+				if err != nil {
+					return err
+				}
 
-			fmt.Fprintf(stderr, "%s Initialized repository in './%s/'\n", utils.GreenCheck(), path)
+				fmt.Fprintf(stderr, "%s Initialized repository in './%s/'\n", utils.GreenCheck(), path)
+			}
 		}
+		return nil
 	}
+	fmt.Fprintln(opts.IO.Out, "Discarding...")
 	return nil
 }
+
+func interactiveRepoCreate(isDescEmpty bool, isVisibilityPassed bool, repoName string) (string, string, string, error) {
+	qs := []*survey.Question{}
+
+	repoOwnerQuestion := &survey.Question{
+		Name: "repoOwner",
+		Prompt: &survey.Input{
+			Message: "Repository name",
+			Default: repoName,
+		},
+	}
+	qs = append(qs, repoOwnerQuestion)
+
+	if isDescEmpty {
+		repoDescriptionQuestion := &survey.Question{
+			Name: "repoDescription",
+			Prompt: &survey.Input{
+				Message: "Repository description",
+			},
+		}
+
+		qs = append(qs, repoDescriptionQuestion)
+	}
+
+	if !isVisibilityPassed {
+		repoVisibilityQuestion := &survey.Question{
+			Name: "repoVisibility",
+			Prompt: &survey.Select{
+				Message: "Visibility",
+				Options: []string{"Public", "Private", "Internal"},
+			},
+		}
+		qs = append(qs, repoVisibilityQuestion)
+	}
+
+	answers := struct {
+		RepoOwner       string
+		RepoDescription string
+		RepoVisibility  string
+	}{}
+
+	err := prompt.SurveyAsk(qs, &answers)
+
+	if err != nil {
+		return "", "", "", err
+	}
+
+	return answers.RepoOwner, answers.RepoDescription, strings.ToUpper(answers.RepoVisibility), nil
+
+}
+
+func confirmSubmission(repoName string, repoOwner string, isConfirmFlagPassed *bool) (bool, error) {
+	qs := []*survey.Question{}
+
+	promptString := ""
+	if repoOwner != "" {
+		promptString = fmt.Sprintf("This will create '%s/%s' in your current directory. Continue? ", repoOwner, repoName)
+	} else {
+		promptString = fmt.Sprintf("This will create '%s' in your current directory. Continue? ", repoName)
+	}
+
+	confirmSubmitQuestion := &survey.Question{
+		Name: "confirmSubmit",
+		Prompt: &survey.Confirm{
+			Message: promptString,
+			Default: true,
+		},
+	}
+	qs = append(qs, confirmSubmitQuestion)
+
+	answer := struct {
+		ConfirmSubmit bool
+	}{}
+
+	err := prompt.SurveyAsk(qs, &answer)
+	if err != nil {
+		return false, err
+	}
+
+	return answer.ConfirmSubmit, nil
+}
+
+func getVisibility() (string, error) {
+	qs := []*survey.Question{}
+
+	getVisibilityQuestion := &survey.Question{
+		Name: "repoVisibility",
+		Prompt: &survey.Select{
+			Message: "Visibility",
+			Options: []string{"PUBLIC", "PRIVATE", "INTERNAL"},
```

Comment:
should also become Public, Private, Internal

Reply:
Agreed